src/test/java/uk/gov/hmcts/ethos/replacement/docmosis/service/RespondNotificationServiceTest.java

```diff
@@ -140,22 +276,90 @@ void testGetRespondNotificationMarkdownAllFields() {
         String result = respondNotificationService.getNotificationMarkDown(caseData);
 
         String expected = "|  | |\r\n"
-                + "| --- | --- |\r\n"
-                + "| Subject | title |\r\n"
-                + "| Notification | [Hearing, Case management orders / requests, Judgment] |\r\n"
-                + "| Hearing | 1: Hearing - Barnstaple - 16 May 2022 01:00 |\r\n"
-                + "| Date Sent | 01-JAN-1970 |\r\n"
-                + "| Sent By | TRIBUNAL  |\r\n"
-                + "| Case management order request | Request |\r\n"
-                + "| Response due | no |\r\n"
-                + "| Party or parties to respond | Both parties |\r\n"
-                + "| Additional Information | info |\r\n"
-                + "| Description | title |\r\n"
-                + " | Document | <a href=\"/documents/6fbf9470-f735-484a-9790-5b246b646fe2/binary\""
-                + " target=\"_blank\">TEST.PDF</a>\r\n"
-                + "| Case management order made by | Judge |\r\n"
-                + "| Name | John Doe |\r\n"
-                + "| Sent to | Both parties |\r\n";
+            + "| --- | --- |\r\n"
+            + "| Subject | title |\r\n"
+            + "| Notification | [Hearing, Case management orders / requests, Judgment] |\r\n"
+            + "| Hearing | 1: Hearing - Barnstaple - 16 May 2022 01:00 |\r\n"
+            + "| Date Sent | 01-JAN-1970 |\r\n"
+            + "| Sent By | TRIBUNAL  |\r\n"
```

Comment:
The figma shows this as "Tribunal" not "TRIBUNAL". QA will deffo raise this and it does look a little bit....aggressive? on the actual page